src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubBuildStatusNotification.java

```diff
@@ -137,7 +137,7 @@ private static void createBuildCommitStatus(Run<?,?> build, TaskListener listene
      * Returns the GitHub Repository associated to a Job.
      *
      * @param job A {@link Job}
-     * @return A {@link GHRepository} or null, either if a scan credentials was not provided, or a GitHubSCMSource was not defined.
+     * @return A {@link GHRepository} or null, either if a scan credentials was not provided, or a GitHubSCMSource was not defined or if notifications are disabled.
```

Comment:
Maybe NIT, but I feel this sentence is starting to be unclear. IIUC, expected is somehow

> A {@link GHRepository}. Will return null if either a scan credentials was not provided, or if a GitHubSCMSource was not defined or if notifications are disabled.

It would clarify that the null is for the end of the sentence, is it?